README.md

```diff
@@ -12,7 +15,7 @@ To build this fork, use `./gradlew zipAll`. The archives of the OLTPBenchmark ap
 
 Please consult the documentation of the original repository on how to further use OLTPBenchmark: [oltpbench/wiki](https://github.com/oltpbenchmark/oltpbench/wiki).
 
-### Supported Benchmarks for Polypheny-DB
+### Supported Benchmarks for Polypheny
 * [TPC-C](./config/polypheny/tpcc.xml)\
```

Comment:
The folder `polypheny` will not exist. Maybe we can write sth. that makes clear that the user has to choose between the folders `polypheny-db` and `polypheny-fram`.